File: python3_solution_set0/9. Palindrome Number.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        
        if x < 0:
            return False
        
        else:
            res = [int(u) for u in str(x)]
            #print(res)
            
            temp = res[::-1]
            for x in range(len(res)):
                if res[x]!=temp[x]:
                    return False
                
                
            return True

File: go_solution_set1/9. Palindrome Number.go
import(
    "fmt"
    "strconv"
)
func reverse_str(str string) string {
    rune_arr := []rune(str)
    var rev []rune
    for i := len(rune_arr) - 1; i >= 0; i-- {
        rev = append(rev, rune_arr[i])
    }
    return string(rev)
}
func isPalindrome(x int) bool {
    if (x < 0){
        return false
    }

    temp := strconv.Itoa(x)
    temp2 := reverse_str(temp)

    return temp == temp2
}
